docs/docs/configuration/connecting-to-data-stores/jaeger.md

```diff
@@ -0,0 +1,102 @@
+# Jaeger
+
+If you want to use Jaeger as the trace data store, you can configure Tracetest to fetch trace data from Jaeger.
+
+You'll configure the OpenTelemetry Collector to receive traces from your system and then send them to Jaeger. And, you don't have to change your existing pipelines to do so.
+
+:::note
+It is important to notice that this relies on the [probabilistic_sampler](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/probabilisticsamplerprocessor) processor, which, at the moment, is only available in the [contrib](https://github.com/open-telemetry/opentelemetry-collector-contrib/) version of the collector.
+:::
```

Comment:
You can remove this. This is not true, only `tail_sampling` is required but just for sending traces directly to Tracetest